<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Tickets</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>971</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="Modifier les tickets" stdset="0">
     <string/>
    </property>
    <property name="page" stdset="0">
     <string/>
    </property>
    <property name="Page" stdset="0">
     <string/>
    </property>
    <property name="title" stdset="0">
     <string notr="true"/>
    </property>
    <property name="tab_ticket" stdset="0">
     <string/>
    </property>
    <property name="Ajouter une ticket" stdset="0">
     <string/>
    </property>
    <widget class="QWidget" name="Ajouter">
     <property name="page" stdset="0">
      <string/>
     </property>
     <property name="Ajouter une ticket" stdset="0">
      <string/>
     </property>
     <attribute name="title">
      <string>Ticket</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>951</width>
        <height>421</height>
       </rect>
      </property>
      <property name="title">
       <string>Création d'une ticket</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Prix</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Horaire arr</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Horaire entr</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_ID">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_Prix">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_Horaire_entr">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_Horaire_arr">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter_4">
       <property name="geometry">
        <rect>
         <x>104</x>
         <y>140</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter une Ticket</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_ID_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>180</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_Prix_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>210</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="Ajouter une ticket" stdset="0">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_Horaire_entr_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>240</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_Horaire_arr_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>270</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_modifier_5">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>300</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Mettre à jour</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>180</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>210</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Prix</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>240</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Horaire entr</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>270</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Horaire arr</string>
       </property>
      </widget>
      <widget class="QTableView" name="tab_ticket_2">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>20</y>
         <width>421</width>
         <height>301</height>
        </rect>
       </property>
       <property name="afficher" stdset="0">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_supp_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>300</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_recherche_ID_2">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>320</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Recherche ID</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>320</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_recherche_PRIX_2">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>350</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Recherche prix</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>350</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_recherche_horaire_entr_2">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>380</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>recherche horaire</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>380</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="on_pb_trier_ticket">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>330</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Tri</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>360</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>par id</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>par prix</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>par horaire</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="ticketVIP">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>360</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>ticket VIP</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_QR_code">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>330</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>QrCode</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_qrcode">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>40</y>
         <width>181</width>
         <height>241</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>300</y>
         <width>201</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>stats</string>
     </attribute>
     <widget class="QCustomPlot" name="plot" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>721</width>
        <height>491</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
